### ISSUE TITLE
Demo Data

### ISSUE THREAD

**Issue by nelsonic** [healthlocker/healthlocker#339]
Each time we do a demo we have to insert a bunch of demo data.
This is tedious repetition that can _easily_ be avoided by using "seed" data which is only applied in the test/staging environment. 
@katbow please investigate adding demo data `before` adding any more features
so that each time we make a `commit` on the project our UI tests run with the demo data and we build up a "_realistic_" view of the application.
Let me know if you need help with this. thanks! 👍

**Comment by nelsonic** [comment]
Hi @katbow, 
this PR is looking good _so far_. 👍 
using the elixir "seeds" (`priv/repo/seeds.exs`) is the "_right_" way of loading "initial" (or our case "Demo") data into the PostgreSQL table(s). ✅ 

_However_, we need _considerably_ more data to be loaded to make the demo look good during demos.
+ in `priv/repo/seeds.exs` we need a _function_ to insert data in bulk.
+ we need a _script_ to _generate_ demo data e.g. using `"faker"`: https://github.com/igas/faker/blob/master/USAGE.md

e.g:
+ 200 service users (_at least 2 service users should have known email address and password_)
+ At least one post per user. 
+ 10 clinicians (
+ 20 care/support worker (

## tl;dr

Related side note: we need to _clarify_ our "_workflow_" regarding [_validating_](https://github.com/dwyl/contributing#part-2-validate-the-need-user-story-or-issue-exists) our understanding of requirements and [estimating](https://github.com/dwyl/contributing#part-3-do-the-work) `before` starting tasks.

**Comment by katbow** [comment]
@nelsonic for 

>  At least one post per user

do you mean *any* type of post, or do you mean posts that are user-specific i.e. they only show up on that users toolkit, like goals and coping strategies?

For stories & tips, I was just going to do a bulk insert where it didn't really matter which user_id it's associated with. With coping strategies & goals, I was going to make 1-2 of each for every user.

**Comment by nelsonic** [comment]
@katbow if it's not _too_ much work please create a function to post one of _each_ type of post. 👍
